holidays/calendars/sinhala_astronomy.py

```
@@ -0,0 +1,221 @@
+    search_end = ts.tt_jd(sunrise.tt + NEW_MOON_SEARCH_DAYS)
+
+    times, phases = almanac.find_discrete(
+        search_start,
+        search_end,
+        almanac.moon_phases(eph),
+    )
+
+    prev_new_moon = None
+    next_new_moon = None
+
+    for t, moon_phase in zip(times, phases):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)